### DISCUSSION TITLE
Thanks for the ongoing support and dev

### DISCUSSION THREAD

**Discussion by andystewart999** [custom-components/ble_monitor#455] [General]
Hi @Ernst79 and @Magalex2x14, I just wanted to say thanks for your ongoing, and regular updates and work on this component!

I had massive issues with the built-in Xiaomi integration, as I'm sure we all did, so you guys and of course the other contributors taking it on, making massive improvements and now adding more devices and features is just one example of the HA community doing what it does best.

So, thanks again!
Andy

**Comment by Ernst79** [comment]
Thanks for the kind words. Much appreciated!! I will change the name of MagAlex to @Magalex2x14 in your post, to address the correct person :wink

**Reply by Magalex2x14** [reply]
This guy has been pinged by accident many times already :)
